### ISSUE TITLE
Cardinality of cbc:ID in cac:PayeeFinancialAccount and cac:FinancialInstitutionBranch is not enforced

### ISSUE THREAD

**Issue by wartab** [CenPC434/validation#505]
https://docs.peppol.eu/poacc/billing/3.0/syntax/ubl-invoice/cac-PaymentMeans/cac-PayeeFinancialAccount/
https://docs.peppol.eu/poacc/billing/3.0/syntax/ubl-invoice/cac-PaymentMeans/cac-PayeeFinancialAccount/cac-FinancialInstitutionBranch/

According to the documentation, these two tags should contain exactly one cbc:ID.

However, both of these rules are not enforced when passing something like this:
```xml
    <cac:PaymentMeans>
        <cbc:PaymentMeansCode name="UNCL1001">60</cbc:PaymentMeansCode>
        <cac:PayeeFinancialAccount>
            <!-- Notice the absence of <cbc:ID> here. -->
            <cac:FinancialInstitutionBranch>
                <cac:FinancialInstitution> <!-- Note that this triggers just a warning. The ID tag should be one level higher -->
                    <cbc:ID>CITIBEBX</cbc:ID>
                </cac:FinancialInstitution>
            </cac:FinancialInstitutionBranch>
        </cac:PayeeFinancialAccount>
    </cac:PaymentMeans>
```

**Comment by phax** [comment]
According to the EN, the BT-86 ("Payment service provider ID"), mapped to `cac:FinancialInstitutionBranch/cbc:ID` is an optional element.

Same as with the other - maybe Peppol rules are stricter

**Comment by wartab** [comment]
`cac:FinancialInstitutionBranch/cbc:ID` is optional, but if `cac:FinancialInstitutionBranch` is present, it should contain a child with `cbc:ID`. 
Currently, you can have `cac:FinancialInstitutionBranch` and as long as you add something else (in this case `<cac:FinancialInstitution>`), it will not enforce the presence of the `cbc:ID`.

**Comment by phax** [comment]
This is what I see in my UBL syntax binding:

<img width="990" height="127" alt="Image" src="https://github.com/user-attachments/assets/a15ec78f-51bb-4ce1-9a9e-4a7a12690d9d" />

Als the EN itself says optional:

<img width="1349" height="99" alt="Image" src="https://github.com/user-attachments/assets/2d6c27f8-a555-46ec-88b3-66f6e7af92d0" />

So the ID is optional to my understanding
